scripts/lib.sh

```diff
@@ -16,15 +16,15 @@ function open_channel() {
     mine_blocks 10
     LND_PUBKEY="$($FM_LNCLI getinfo | jq -e -r '.identity_pubkey')"
     $FM_LIGHTNING_CLI connect $LND_PUBKEY@127.0.0.1:9734
-    until $FM_LIGHTNING_CLI -k fundchannel id=$LND_PUBKEY amount=0.1btc push_msat=5000000000; do sleep $POLL_INTERVAL; done
+    until $FM_LIGHTNING_CLI -k fundchannel id=$LND_PUBKEY amount=0.1btc push_msat=5000000000; do sleep $FM_POLL_INTERVAL; done
     mine_blocks 10
-    until [[ $($FM_LIGHTNING_CLI listpeers | jq -e -r ".peers[] | select(.id == \"$LND_PUBKEY\") | .channels[0].state") = "CHANNELD_NORMAL" ]]; do sleep $POLL_INTERVAL; done
+    until [[ $($FM_LIGHTNING_CLI listpeers | jq -e -r ".peers[] | select(.id == \"$LND_PUBKEY\") | .channels[0].state") = "CHANNELD_NORMAL" ]]; do sleep $FM_POLL_INTERVAL; done
 }
 
 function await_bitcoin_rpc() {
     until $FM_BTC_CLIENT getblockchaininfo 1>/dev/null 2>/dev/null ; do
         >&2 echo "Bitcoind rpc not ready yet. Waiting ..."
-        sleep "$POLL_INTERVAL"
+        sleep "$FM_POLL_INTERVAL"
     done
 }
```

Comment:
I wish we had #665, then we could be fancy here without too much overhead/copy paste and just add one period every second and not print a new line.

Reply:
I tried implementing this, but don't know bash well enough, and that frustration got me to try https://github.com/fedimint/fedimint/pull/1878.